im.pidgin.Pidgin3.yml

```diff
@@ -0,0 +1,126 @@
+app-id: im.pidgin.Pidgin3
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: pidgin3
+finish-args:
+  # X11 + XShm access
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  # Wayland access
+  - "--socket=wayland"
+  # GPU acceleration if needed
+  - "--device=dri"
+  # Needs to talk to the network:
+  - "--share=network"
+  # Audio
+  - "--socket=pulseaudio"
+  # Notifications (purple-toast)
+  - "--talk-name=org.freedesktop.Notifications"
+  # Accessibility
+  - "--talk-name=org.a11y.Bus"
+  # libsecret plugin
+  - "--talk-name=org.freedesktop.secrets"
+  # kwallet plugin
+  - "--talk-name=org.kde.kwalletd5"
+  - "--talk-name=org.kde.kwalletd6"
+  # Bonjour
+  - "--system-talk-name=org.freedesktop.Avahi"
+  # Plugin Paths
+  - "--env=PURPLE_PLUGIN_PATH=/app/plugins/purple-3"
+  - "--env=PIDGIN_PLUGIN_PATH=/app/plugins/pidgin-3"
+
+add-extensions:
+  im.pidgin.Purple3.Plugin:
```

Comment:
this namespace looks incorrect.

Reply:
How so?

Reply:
`im.pidgin.Purple3` doesn't exist.

Reply:
but see the comment below about having one extension point.